emote/extra/threading.py

```diff
@@ -0,0 +1,87 @@
+import threading
```

Comment:
There's already a LockedResource in `emote/utils` that looks very similar.

Reply:
Ah, it's likely the same.

Reply:
This still shows up when I review it. Did you remove it? I feel like a rebase has gone wrong or something here.